qqqwww123cdv/hometask-3/rainfall.rb

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def data
+  file = File.new('data.txt', 'r:UTF-8')
+  file.read
+end
+
+def weather_data(town, data)
+  data.split("\n").find do |value|
+    if value.include? town
+      monthly_rainfall = value.scan(/\d*\.\d/).map(&:to_f)
+      return monthly_rainfall
+    end
+  end
+end
+
+def mean(town, data)
+  return -1.0 unless data.match "#{town}:"
+
+  average_rainfall = weather_data(town, data)
+  average_rainfall.sum / 12
+end
+
+def variance(town, data)
+  return -1.0 if mean(town, data) == -1
+
+  rainfall_values = mean(town, data)
```

Comment:
same here 
putting this above the `return` will let us get rig of the redundant `mean` call